icd-503-high.yaml

```diff
@@ -0,0 +1,404 @@
+# Based on the ICD-503 High-High-High Security Controls
+# Current as of 15-June-2018
+# Maintained by Mark Shoger (mshoger@redhat.com)
+# https://www.dni.gov/files/documents/ICD/ICD_503.pdf
+name: ICD503-High
```

Comment:
``name`` is a human readable field. Would ``ICD 503 High`` be appropriate? Or ``ICD 503 High-High-High``? I don't know what the appropriate nomenclature would be.

Reply:
(same applies to the ``moderate`` and ``low`` profiles)